modules/ROOT/pages/salesforce-releases-migrations.adoc

```diff
@@ -0,0 +1,24 @@
+= Salesforce Releases and Migrations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+The Salesforce organization dedicated to your API community follows the standard https://trailhead.salesforce.com/en/content/learn/modules/sf_releases/sf_releases_start[Salesforce release process]. This automatically upgrades your organization to the latest Salesforce release version three times each year. As an organization administrator, you'll receive email notifications about the changes at least three months in advance, including any required steps for you to follow, guidance, and other resources.
+
+If any additional steps are required to prepare Anypoint API Community Manager, they will be provided from your MuleSoft customer success representative and on https://status.mulesoft.com/[trust.mulesoft.com]. Many Salesforce upgrades do not require any additional steps for API Community Manager.
+
+== Salesforce Organization Migrations
```

Comment:
This is organizationally odd. The parent topic is about releases and migrations, and fully explains the release preparation process but not the migration process...which has become a subtopic:

= Releases and Migrations
== Migrations
== Getting Help

However, the migration text states that you typically experience no impact, but MuleSoft will let you know.
Then Getting Help repeats what the previous two topics say about Salesforce and MuleSoft help.

This topic could easily be just one text, titled something like Preparation for Salesforce Releases and Migrations.